salon_creator/lib/screens/login_page.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:salon_creator/authentication/sign_in.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(children: [
+        new Container(
+          decoration: new BoxDecoration(
+            image: new DecorationImage(
+              image: new AssetImage("assets/background.jpg"),
+              fit: BoxFit.fill,
+            ),
+          ),
+        ),
+        Container(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Center(
+                child: Container(
+                  width: 300,
+                  child: TextButton(
+                    style: TextButton.styleFrom(primary: Colors.transparent),
+                    onPressed: () async {
+                      await signInWithGoogle();
+                      if (FirebaseAuth.instance.currentUser != null) {
+                        addUserToDatabase();
+                        Navigator.of(context).pushNamed('/home');
+                      }
+                    },
+                    child: Image.asset(
```

Comment:
I think we should display Login text instead of an image for adaptive rendering and fast screen loads.